include/seahorn/Expr/ExprOpSort.hh

```diff
@@ -18,36 +17,52 @@ enum class SimpleTypeOpKind {
   UNINT_TY,
   ARRAY_TY,
   STRUCT_TY,
-  ANY_TY
+  ANY_TY,
+  ERROR_TY
 };
+
+namespace typeCheck {
+  namespace simpleType {
+struct Simple {
+  static inline Expr inferType(Expr exp, TypeChecker &tc) {
+    return exp;
```

Comment:
type of a type is `ANY`, not type itself. For example, the type of `int` is not `int`, it is `type int`. But we don't need the hierarchy that deep, so we can conflate it into `any`